### DISCUSSION TITLE
Galaxy brain badge 1

### DISCUSSION THREAD

**Discussion by nel0x** [Schweinepriester/github-profile-achievements#57] [Q&A] [answered]
Lorem ipsum

**Comment by melvincwng** [comment]
Hi @nel0x, pardon me for asking, are you trying to get the Galaxy Brain badge?

**Comment by melvincwng** [comment, marked as answer]
@nel0x, glad that your issue has been resolved!

Kindly 'Mark as answer' or close this discussion, so that the rest are aware this issue has been addressed :)

**Comment by nel0x** [comment]
for everyone else see https://github.com/Schweinepriester/github-profile-achievements/discussions/58
